flows/vision-zero/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get update && \
     ln -fs /usr/share/zoneinfo/America/Chicago /etc/localtime && \
     dpkg-reconfigure -f noninteractive tzdata
 RUN apt-get upgrade -y
-RUN apt-get install -y vim magic-wormhole aptitude python3-pip docker.io rsync p7zip-full black poppler-utils tesseract-ocr wget postgresql-client pgloader postgresql-common libpq-dev
+RUN apt-get install -y vim magic-wormhole aptitude python3-pip docker.io rsync p7zip-full black poppler-utils tesseract-ocr wget postgresql-client pgloader postgresql-common libpq-dev openssh-server
```

Comment:
The dockerized-agent container is actually the bastion host for the local DB in the local stack here 🪄